index.js

```diff
@@ -311,7 +312,7 @@ module.exports = class HyperDHT extends DHT {
       }
     })
     await query.finished()
-    return { signature, seq }
+    return { closestNodes, signature, seq }
```

Comment:
closestNodes: query.closestNodes

Reply:
hmm gives build error, is `query` from dht-rpc?
https://github.com/mafintosh/dht-rpc/blob/master/lib/query.js

Reply:
Otherwise you can link the error here if you want soem help.

Reply:
This is the error:
https://github.com/hyperswarm/dht/pull/51/checks?check_run_id=3534495624

Reply:
`  D:\a\dht\dht\index.js:315:19: Parsing error: Unexpected token .
npm ERR! Test failed.  See above for more details.
Error: Process completed with exit code 1.`

Reply:
I think `query` is a `streamx` readable stream...